WebRoot/css/zm.css

```diff
@@ -1711,6 +1745,14 @@ INPUT[type="text"].search_input-expanded:focus {
 	line-height:normal;
 }
 
+.EditorHolder {
+	border-width: 0px !important;
+}
+
+.attBubbleContainer {
+	border: none;
```

Comment:
The border are removed as part of another commit for bug 787.
So removing them.